declarations/options.d.ts

```diff
@@ -0,0 +1,66 @@
+/** @typedef {import("eslint").ESLint.Options} ESLintOptions */
+/** @typedef {import('eslint').ESLint.LintResult} LintResult */
+/** @typedef {import('eslint').ESLint.LintResultData} LintResultData */
+/**
+ * @callback FormatterFunction
+ * @param {LintResult[]} results
+ * @param {LintResultData=} data
+ * @returns {string}
+ */
+/**
+ * @typedef {Object} OutputReport
+ * @property {string=} filePath
+ * @property {string|FormatterFunction=} formatter
+ */
+/**
+ * @typedef {Object} Options
+ * @property {string=} context
+ * @property {boolean=} emitError
+ * @property {boolean=} emitWarning
+ * @property {string=} eslintPath
+ * @property {boolean=} failOnError
+ * @property {boolean=} failOnWarning
+ * @property {string|string[]=} files
+ * @property {string|string[]=} extensions
+ * @property {boolean=} fix
+ * @property {string|FormatterFunction=} formatter
+ * @property {boolean=} lintDirtyModulesOnly
+ * @property {boolean=} quiet
+ * @property {OutputReport=} outputReport
+ */
+/**
+ * @param {Options} pluginOptions
+ * @returns {Options}
+ */
+export function getOptions(pluginOptions: Options): Options;
+/**
+ * @param {Options} loaderOptions
+ * @returns {ESLintOptions}
+ */
+export function getESLintOptions(loaderOptions: Options): ESLintOptions;
+export type ESLintOptions = import('eslint').ESLint.Options;
+export type LintResult = import('eslint').ESLint.LintResult;
+export type LintResultData = import('eslint').ESLint.LintResultData;
```

Comment:
`@types/eslint` should be in `dependencies`

Reply:
Moved